project_singlefamilydetached/singlefamilydetached_with_upgrades.yml

```diff
@@ -0,0 +1,1165 @@
+        costs:
+          - value: 72
+            multiplier: Fixed (1)
+        lifetime: 14
+  - upgrade_name: ENERGY STAR Dishwashers
+    options:
+      - option: Dishwasher|290 Rated kWh, 80% Usage
+        apply_logic: Dishwasher|318 Rated kWh, 80% Usage
+        costs:
+          - value: 1000
+            multiplier: Fixed (1)
+        lifetime: 11
+        reference_scenrio: Standard Dishwasher
+      - option: Dishwasher|290 Rated kWh, 100% Usage
+        apply_logic: Dishwasher|318 Rated kWh, 100% Usage
+        costs:
+          - value: 1000
+            multiplier: Fixed (1)
+        lifetime: 11
+        reference_scenrio: Standard Dishwasher
+      - option: Dishwasher|290 Rated kWh, 120% Usage
+        apply_logic: Dishwasher|318 Rated kWh, 120% Usage
+        costs:
+          - value: 1000
+            multiplier: Fixed (1)
+        lifetime: 11
+        reference_scenrio: Standard Dishwasher
+  - upgrade_name: Standard Dishwashers
+    options:
+      - option: Dishwasher|318 Rated kWh, 80% Usage
+        apply_logic: Dishwasher|318 Rated kWh, 80% Usage
+        costs:
+          - value: 880
+            multiplier: Fixed (1)
+        lifetime: 11
+      - option: Dishwasher|318 Rated kWh, 100% Usage
+        apply_logic: Dishwasher|318 Rated kWh, 100% Usage
+        costs:
+          - value: 880
+            multiplier: Fixed (1)
+        lifetime: 11
+      - option: Dishwasher|318 Rated kWh, 120% Usage
+        apply_logic: Dishwasher|318 Rated kWh, 120% Usage
+        costs:
+          - value: 880
+            multiplier: Fixed (1)
+        lifetime: 11
+  - upgrade_name: ENERGY STAR Refrigerators
+    options:
+      - option: Refrigerator|EF 19.9
+        apply_logic: Refrigerator|EF 17.6
+        costs:
+          - value: 670
+            multiplier: Fixed (1)
+        lifetime: 17.4
+        reference_scenario: Refrigerator Reference
+  - upgrade_name: Refrigerator Reference
+    options:
+      - option: Refrigerator|EF 17.6
+        apply_logic:
+          or:
+            - Refrigerator|EF 6.7
+            - Refrigerator|EF 10.2
+            - Refrigerator|EF 10.5
+            - Refrigerator|EF 15.9
+        costs:
+          - value: 660
+            multiplier: Fixed (1)
+        lifetime: 17.4
+
+##Lighting # 40
+  - upgrade_name: LEDs
+    options:
+      - option: Lighting|100% LED
+        apply_logic:
+          or:
+            - Lighting|100% Incandescent
+            - Lighting|60% CFL Hardwired, 34% CFL Plugin
+            - Lighting|60% LED Hardwired, 34% CFL Plugin
+            - Lighting|60% CFL
+            - Lighting|100% CFL
+            - Lighting|100% LED, Low Efficacy
+        costs:
+        #  - value: 0.006
+        #    multiplier: [lumens]
+          - value: 0.40
+            multiplier: Floor Area, Lighting (ft^2)
+        lifetime: 78.28
+
+
+timeseries_csv_export:
+  reporting_frequency: Hourly
+  include_enduse_subcategories: true
+eagle:
+  account: enduse
```

Comment:
```suggestion
  account: <account you are authorized to use>
```

Reply:
This is probably a good idea. I may include this in all yml files in this repo.